ReactAndroid/src/test/java/com/facebook/react/modules/network/NetworkingModuleTest.java

```diff
@@ -309,6 +309,55 @@ public Object answer(InvocationOnMock invocation) throws Throwable {
     assertThat(requestHeaders.get("User-Agent")).isEqualTo("React test agent/1.0");
   }
 
+  @Test
+  public void testPostJsonContentTypeHeader() throws Exception {
```

Comment:
I copied this from previous tests. It's possible more of the mocks below can be removed.